src/domain/app/i18n/sv.json

```diff
@@ -560,6 +561,7 @@
         "general": "Om antalet deltagare i evenemanget är begränsat, ange lägsta och högsta antal deltagare här.",
         "volunteering": "Om antalet deltagare i volontärarbetet är begränsat, ange lägsta och högsta antal deltagare här."
       },
+      "infoTextAttendeeCapacityUserUnknown": "If the number of event participants is not limited in any way, write a rough estimate of the maximum number of participants. (This information will be used for statistical purposes).",
```

Comment:
Should this be in Swedish?

Reply:
Translations: https://github.com/City-of-Helsinki/linkedcomponents-ui/pull/185/commits/9d8ee2af93331ff695811a6a1d684ae18200167c